index.html

```diff
@@ -104,7 +118,7 @@
             <div>
                 <img src="misc/logo.png" height="64">
                 <h5>Warframe Drop Data</h5>
-                <a target="_blank" href="https://github.com/WFCD/warframe-drop-data#api-endpoints">API Documentation</a> | <a target="_blank" href="https://github.com/WFCD/warframe-drop-data">Github</a>
+                <a href="https://drops.warframestat.us/diff.html">Compare Versions</a> | <a target="_blank" href="https://github.com/WFCD/warframe-drop-data#api-endpoints">API Documentation</a> | <a target="_blank" href="https://github.com/WFCD/warframe-drop-data">Github</a>
```

Comment:
link to `./diff.html` instead should work better for the compare versions
that way it's domain agnostic

Reply:
Done.